manifests/config.pp

```diff
@@ -40,7 +40,6 @@
   } else {
     $_server_conf_dir = "${keycloak::install_base}/domain/servers/${keycloak::server_name}/configuration"
     $_add_user_keycloak_args = "--user ${keycloak::admin_user} --password ${keycloak::admin_user_password} --realm master --sc ${_server_conf_dir}/" # lint:ignore:140chars
-    $_subdir = 'domain'
```

Comment:
If `$_subdir` is no longer needed, would need to also remove that variable from the first part of this condition.